src/main/java/com/minecolonies/compatibility/tinkers/SlimeTreeProxy.java

```diff
@@ -0,0 +1,43 @@
+package com.minecolonies.compatibility.tinkers;
+
+import net.minecraft.block.Block;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * This is the fallback for when tinkers is not present!
+ */
+public class SlimeTreeProxy
+{
+    /**
+     * This is the fallback for when tinkers is not present!
+     *
+     * @param block the block.
+     * @return if the block is a slime block.
+     */
+    protected boolean checkForTinkersSlimeBlock(@NotNull final Block block)
+    {
+        return false;
+    }
+
+    /**
+     * This is the fallback for when tinkers is not present!
+     *
+     * @param block the block.
+     * @return if the block is a slime leaf.
+     */
+    protected boolean checkForTinkersSlimeLeaves(@NotNull final Block block)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "block". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1172)